generators/chipyard/src/main/scala/ConfigFragments.scala

```diff
@@ -193,7 +193,20 @@ class WithTLBackingMemory extends Config((site, here, up) => {
   case ExtTLMem => up(ExtMem, site) // enable TL backing memory
 })
 
-class WithTileFrequency(fMHz: Double) extends ClockNameContainsAssignment("core", fMHz)
+class WithSerialTLBackingMemory extends Config((site, here, up) => {
+  case ExtMem => None
+  case SerialTLKey => up(SerialTLKey, site).map { k => k.copy(
+    memParams = {
+      val memPortParams = up(ExtMem, site).get
+      require(memPortParams.nMemoryChannels == 1)
+      memPortParams.master
+    },
+    isMemoryDevice = true
+  )}
+})
+
+class WithTileFrequency(fMHz: Double) extends ClockNameContainsAssignment("tile", fMHz)
```

Comment:
Instead of adding another fragment, maybe add an optional parameter which, if set, specifies a hart, if not set, specifies all harts?